metis.py

```
@@ -0,0 +1,272 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2015 ysitu <ysitu@users.noreply.github.com>
+# All rights reserved
+
+"""
+Wrappers of METIS graph partitioning functions.
+"""
+
+from contextlib import contextmanager
+from itertools import chain
+from sys import exc_info
+
+import networkx as nx
+from networkx.utils import not_implemented_for
+from decorators import convert_exceptions
+from _types import MetisError
+
+try:
+    from _metis import node_nd, part_graph, compute_vertex_separator
+    from _types import MetisOptions
+    __all__ = ['node_nested_dissection', 'partition', 'vertex_separator',
+               'MetisOptions']
+except ImportError:
+    __all__ = []
```

After a success installation, these imports should just work. Let's not hide any errors.